src/builders.jl

```diff
@@ -263,7 +263,7 @@ function build_readme_docs(pkgname, pkgroot, docsdir, mod, src_prefix, href_pref
     end
 
     @eval Module() begin
-        using Documenter
+        using ..DocumentationGenerator.Documenter
```

Comment:
I think this should be sufficient:

```suggestion
        const Documenter = $Documenter
        using .Documenter
```